### PR TITLE
Add duplicate ticket feature

### PR DESCRIPTION
**Pull request by timja**
Fixes https://github.com/hmcts/slack-help-bot/issues/50
Fixes https://github.com/hmcts/slack-help-bot/issues/27

Uses chatops to not complicate the UI:
```
@PlatOpsHelp help
@PlatOpsHelp duplicate DTSPO-111
```

image

image

image

Adds links to Jira:
image

And marks as Done:
image

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -60,28 +50,22 @@ const reportChannelId = config.get('slack.report_channel_id');
...
         }
-        myResponse = {
+        const myResponse = {
             status: "UP",
             slack: {
-                connection: (app.receiver.client.badConnection ? "DOWN" : "UP"),
-                status: (isSlackOkay ? "UP" : "DOWN"),
-            },
-            jira: {
-                status: (isJiraOkay ? "UP" : "DOWN"),
```
Comment (timja) [inline app.js:84]: this was just hardcoded to be up so I've removed the dead code

**Review by cakeben — APPROVED** [review]

File: app.js (hunk 1 above)
Comment (ssian2) [reply app.js:84]: It's not hardcoded. Those flags can get set to false when posting to slack or JIRA fails in the new workflow

Comment (timja) [reply app.js:84]: hmm I don't think that's the best approach.

that doesn't mean Jira is down, a single failed request shouldn't cause a healthcheck failure.

fine to alert via app insights that there's failures

Comment (ssian2) [reply app.js:84]: Okay, might be worth re-visiting what's reported on the health page, then

**Review by ssian2 — APPROVED** [review]

**Pull request merged**